### DISCUSSION TITLE
Trouble getting started

### DISCUSSION THREAD

**Discussion by jporemski** [RhetTbull/osxphotos#852] [Q&A] [answered]
I am trying to get started using osxphotos but can't seem to get passed the initial stages. Everything I type into Terminal comes back with an error. Can someone please give me some hints on what I am doing wrong (see examples below)? I am confident that once I get passed this initial hump I'll be able to understand the rest of the syntax.

I've attached information for my computer

Thanks very much.

***Example Terminal session***

Last login: Mon Nov 28 12:45:48 on ttys000
JXXX@Johns-Air ~ % osxphotos export --export-by-date ~/Pictures/Photos\ Photos Library_Temp.photoslibrary ~/Desktop/export

zsh: command not found: osxphotos
JXXX@Johns-Air ~ % >osxphotos export --export-by-date ~/Pictures/Photos\ Photos Library_Temp.photoslibrary ~/Desktop/export

export: bad option: -e
JXXX@Johns-Air ~ % >osxphotos export ~/Pictures/Photos\ Photos Library_TEMP.photoslibrary ~/Desktop/export

export: not valid in this context: /Users/JPorensky/Pictures/Photos Photos
JXXX@Johns-Air ~ % >osxphotos export ~/Desktop\ export --export-by-date

export: not valid in this context: /Users/JPorensky/Desktop export
JXXX@Johns-Air ~ % >osxphotos export ~/Desktop\ export \

export: not valid in this context: /Users/JPorensky/Desktop export
JXXX@Johns-Air ~ % >osxphotos export Desktop
JXXX@Johns-Air ~ % 
![SystemInfo](https://user-images.githubusercontent.com/119357725/204370028-e5306d3f-813e-4ae4-8fda-71b1d1423ebe.jpg)

**Comment by RhetTbull** [comment]
Looks like osxphotos isn't in your path. How did you install it? If you type `which osxphotos` what is the output?

**Comment by jporemski** [comment, marked as answer]
Thank you very much for your quick response. This is what I get when I type
"which osxphotos"

Last login: Mon Nov 28 12:48:43 on ttys000

***@***.*** ~ % which osxphotos

osxphotos not found


I followed the instructions for installation using "PIPX." This is what I
get when I run the command to update:

***@***.*** ~ % pipx upgrade osxphotos

osxphotos is already at latest version 0.54.4 (location:

/Users/JPorensky/.local/pipx/venvs/osxphotos)


Thanks again.

**Reply by RhetTbull** [reply]
Run `pipx ensurepath`, close the Terminal, reopen it and try again. Looks like pipx didn't install itself in the path when it was installed.

**Comment by jporemski** [comment]
That seems to have done the trick. Thank you very much!
